building-with-ocean/using-ocean-libraries/create-datatoken-with-fixed-pricing.md

```diff
@@ -0,0 +1,177 @@
+# Create datatoken with fixed pricing
+
+This tutorial will guide you for creating your own data NFT and a datatoken using Ocean libraries. To know more about data NFT and datatoken please refer [this page](../../core-concepts/datanft-and-datatoken.md). Ocean Protocol supports different pricing schemes which can be set while publishing an asset. Please refer [this page](../../core-concepts/asset-pricing.md) for more details on pricing schemes.
+
+#### Configuration
```

Comment:
I think this could be changed to prerequisites and then expanded to a short bullet point list.